app/views/keywords/_keywords_table.erb

```diff
@@ -0,0 +1,29 @@
+<div class="keywords-table">
+  <table class="table">
+    <thead class="thead-light">
+      <tr>
+        <th scope="col">ID</th>
+        <th scope="col">Keyword</th>
+        <th scope="col">Status</th>
+        <th scope="col">Uploaded at</th>
+        <th scope="col"></th>
+      </tr>
+    </thead>
+
+    <tbody class="keywords-table__body">
+      <% presenters.map do |presenter| %>
+        <tr data-keyword-id="<%= presenter.keyword.id %>">
+          <td><%= presenter.keyword.id %></td>
+          <td><%= presenter.keyword.content %></td>
+          <td>
+            <span class="<%= presenter.status_badge %>">
+              <%= presenter.keyword.status.capitalize %>
+            </span>
+          </td>
+          <td><%= presenter.keyword.created_at.strftime("%H:%M, %-d %B") %></td>
```

Comment:
Such data presentation should be handled in the presenter class

Reply:
Fixed in 94d7cb985bdf7dd93341a1a0a777eed167fcb235